03-FLEX/Portfolio/style/style.css

```diff
@@ -0,0 +1,282 @@
+@import "normalize.css";
+@import "colors.css";
+
+body {
+    font-family: 'Vazirmatn', sans-serif;
+    margin: 0;
+
+    background-color: rgb(245, 245, 245);
+    color: var(--dark-blue);
+}
+
+li {
+    list-style: none;    
+}
+
+.container {
+    width: 100%;
+    max-width: 1000px;
+    margin: 0 auto;
+}
+
+.hr-line {
+    height: 2px;
+    background-color: lightgrey;
+    border: none;
+    margin: 50px 0;
+}
+
+
+/* Header */
+
+.header {
+    background-color: var(--main-blue);
+}
+
+.header__logo {
+    padding-right: 100px;
+}
+
+.header-nav {
+    display: flex;
+    justify-content:space-between;
+    align-items: center;
+}
+
+.header-list {
+    height: 30px;
+
+    color: white;
+
```

Comment:
try removing empty spaces like this. Basically you separate each selector block with spaces, don't need to add any spaces inside them.